bin/tuning.py

```diff
@@ -56,28 +54,39 @@ def SleepRayTune(config, cfg: DictConfig, num_epochs, num_gpu):
 
 
 @hydra.main(config_path="../config", config_name="config")
-# model=m2, dataset=m2, training=m2 in terminal
-def main(cfg: DictConfig, num_epochs=100, gpus_per_trial=1):
+# model=tuning, dataset=m2, training=tuning optimizer=m2 in terminal
```

Comment:
For now, this script has to be run with `python tuning.py model=tuning dataset=m2 training=tuning optimizer=m2`. Would it be better to override all or some of them in the script, since they're mostly default settings? So that it can be run with `python tuning.py model=some_model_to_tune` 
@zzhaozhe-profolio

Reply:
I think it's fine to keep as is since we might want to tune other models that are not `m2` later on.